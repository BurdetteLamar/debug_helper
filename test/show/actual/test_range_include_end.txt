---
Range (message='test_range_include_end'):
  first: 0
  last: 4
  exclude_end?: false
