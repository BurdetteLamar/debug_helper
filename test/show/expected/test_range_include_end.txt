---
Range (message='test_range_include_end'):
  Range#first: 0
  Range#last: 4
  Range#exclude_end?: false
